src/components/basket-item/basket-item.jsx

```diff
@@ -4,27 +4,69 @@ var React = require('react');
 module.exports = React.createClass({
   propTypes: {
     currencySymbol: React.PropTypes.string,
+    freeText: React.PropTypes.string,
+    removeText: React.PropTypes.string,
     price: React.PropTypes.number,
+    title: React.PropTypes.node,
+    toggleDescription: React.PropTypes.bool,
```

Comment:
@pmcnr-hx this isn't state.. it's a configuration property. We are saying "do you want to toggle description?"

Later we do use state to manage that visibility... that is called `descriptionVisibility` which is pretty similar to your suggestion :)